config/locales/views/notes/stats/en.yml

```diff
@@ -15,6 +15,12 @@ en:
         <a href='%{url2}'>one month</a>, or
         <a href='%{url3}'>one week</a>
         further back.
+      try_time_further_forward: |
```

Comment:
Would you mind adding these to `config/locales/en.yml` as we are seeking to consolidate these -- see https://github.com/publiclab/plots2/issues/3515

Otherwise this is great! Can you upload a screenshot? 📸  Thanks!!!

Reply:
I believe I added a screenshot to my original pull request, but I will attach another one here.
![screenshot from 2018-10-27 21-40-31](https://user-images.githubusercontent.com/44309027/47760439-423db680-dc82-11e8-934a-c15e98b5baa0.png)

Would you like me to move the translations in another PR or commit to this one? Sorry, I am somewhat new to GitHub :( .

Thank you,
Kevin Luo

Reply:
You can work on it here @kevinzluo